API/MoodisAPI/apiMoodis/apiMoodis/Controllers/EmotionController.cs

```diff
@@ -0,0 +1,95 @@
+﻿using apiMoodis.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+using System.Data.Entity;
+
+namespace apiMoodis.Controllers
+{
+    public class EmotionController : ApiController
+    {
+        public HttpResponseMessage Get()
+        {
+            using (DatabaseContext dbContext = new DatabaseContext())
+            {
+                var emotions = dbContext.Emotions.ToList();
+                if (emotions.Count() == 0)
+                {
+                    return Request.CreateErrorResponse(HttpStatusCode.NotFound, "There are no emotions");
+                }
+                else
+                {
+                    return Request.CreateResponse(HttpStatusCode.OK, emotions);
+                }
+            }
+        }
+
+        public HttpResponseMessage Get(string id)
+        {
+            using (DatabaseContext dbContext = new DatabaseContext())
+            {
+                var entity = dbContext.Emotions.FirstOrDefault(emotion => emotion.Id == id);
+                if (entity != null)
+                {
+                    return Request.CreateResponse(HttpStatusCode.OK, entity);
+                }
+                else
+                {
+                    return Request.CreateErrorResponse(HttpStatusCode.NotFound, "Emotion with Id " + id + " Not found");
+                }
+            }
+        }
+
+        public HttpResponseMessage Post([FromBody] Emotion emotion)
```

Comment:
I would suggest:
```csharp
[HttpPost]
public HttpResponseMessage AddEmotion([FromBody]...)
```

This will do the same, but you will have a more descriptive controller method name.